src/test/java/api/support/fixtures/LoanPoliciesFixture.java

```diff
@@ -52,6 +55,21 @@ public IndividualResource create(LoanPolicyBuilder builder) {
     return loanPolicyRecordCreator.createIfAbsent(builder);
   }
 
+  public void create(List<String> ids) {
+    ids.stream()
+      .map(id -> new LoanPolicyBuilder()
+        .withId(UUID.fromString(id))
+        .withName("Example LoanPolicy " + id))
+      .forEach(loanPolicyRecordCreator::createIfAbsent);
+  }
+
+  public void create(UUID id) {
+    loanPolicyRecordCreator.createIfAbsent(
+      new LoanPolicyBuilder()
+      .withId(id)
+      .withName("Example LoanPolicy " + id));
+  }
```

Comment:
What are these methods used for? Could they be defined closer to the tests that use them rather than the fixture?

Reply:
It creates dummy loan policy with specific ID and generated loan policy name.  I can do it closer to the tests, but this method is used in 5 tests. Do you think it's OK to do it in these tests? Won't it jam the tests?

Reply:
Is that 5 tests within the same test class, or spread across multiple classes?

Reply:
It spread across 5 classes

Reply:
Ok, maybe this should be left for now. I would prefer it if there was a better way than adding to the fixture. I don't really understand why the builder cannot be used directly in the tests.

Reply:
OK, I've updated everything. Now I use builder directly in the tests.

Reply:
Ok, I wasn't necessarily expecting that, thank you.